README.md

```diff
@@ -151,6 +151,11 @@ Put each branch one by one in sync with its parent and remote tracking branch.
 git machete advance
 ```
 
+### Synchronize with the remote repository, delete untracked and unmanaged branches, checkout your open PRs into local branches
```

Comment:
Hmm this is a part of GitHub integration (esp. the point about authentication applies)

Regardless, I'd also reconsider which commands really need to be included in this README... I'd actually almost surely remove `git machete github retarget-pr` which is quite specific; `git machete github anno-prs` is probably also unneeded in a starter/teaser text like this README

Reply:
TBD if `git machete clean` really qualifies

Reply:
Sure, i will remove both `git machete github retarget-pr` and `git machete github anno-prs`